include/cling/Interpreter/Interpreter.h

```diff
@@ -712,6 +713,24 @@ namespace cling {
     ///
     void* getAddressOfGlobal(llvm::StringRef SymName, bool* fromJIT = 0) const;
 
+    ///\brief Get a given macro definition by name.
+    ///
+    ///\param[in]  Name - the name of the macro to look for
+    ///
+    ///\returns the MacroInfo if the macro was defined, otherwise null
+    ///
+    const clang::MacroInfo* getMacro(llvm::StringRef Name) const;
```

Comment:
Dunno.
If it were a global function I might agree, but couldn't any future introduction of the functionality you're describing be done using **_command_** instead of **_macro_** and just leave macro to mean what it does in LLVM/clang?